### ISSUE TITLE
Question: what're the default for AdvertiserTrackingEnabled, AdvertiserIDCollectionEnabled, and AutoLogAppEventsEnabled

### ISSUE THREAD

**Issue by JacopoDaeli** [facebook/facebook-ios-sdk#1632]
Hello,

if I do not explicitly set `AdvertiserTrackingEnabled`, `AdvertiserIDCollectionEnabled,` and `AutoLogAppEventsEnabled`.
What are the default values used?

Also, for iOS 14 users, is `SKAdNetwork` used automatically whenever IDFA is not available?
Or is there something explicitly that needs to be configured for SKAdNetwork to be used?

Finally, will `SKAdNetwork` be used if `AdvertiserIDCollectionEnabled` (and/or `AdvertiserIDCollectionEnabled`) has not been set (or set to false)?

In short I am also trying to understand how these flags work together, and what happens if these flag are not set (or set to false).

Thanks

**Comment by joesus** [comment]
Aggregated into #1667
